src/Deserialization/Deserialization.csproj

```diff
@@ -10,4 +11,9 @@
     <ProjectReference Include="..\Representation\Representation.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="libLLVM" Version="12.0.0" />
+    <PackageReference Include="LLVMSharp" Version="12.0.0-beta1" />
```

Comment:
Someone likes to live dangerously. 😜

Reply:
They only provide "stable" packages for libLLVM, not LLVMSharp to allow them to make whatever changes they want, so they always leave them as beta versions. I guess this is correct for strict SemVer reasons.